### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards> InputsWithValidations.json]: Incorrect role as link is defined for the 'Reply' button.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#8188]
### Target Platforms

UWP

### SDK Version

3.0.2210.26001

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS version: 22H2 (Build 25267.1000)
Application: AdaptiveCardVisualizer
Application Version: 3.0.2210.26001
Screen reader: Narrator

**Repro Steps:**

1. Open the Adaptive Card Visualizer application.
2. Navigate to the 'InputsWithValidations.json' through tab key and activate it.
3. Now, navigate to 'Reply' button present beside of 'Text input with an inline action with no icon' edit field through tab key.
4. Observe the issue.

**Actual Result:**
Under 'InputsWithValidations.json', incorrect role as **'Link**' is defined to the 'Reply' button.

**Expected Result:**
Proper and correct role as '**Button**' should be defined for the 'Reply' button.

**Notes:**
Same issue repro for 'View' button present under 'ActivityUpdate.json'.

**User Impact:**
Screen reader user will get impacted as user will not be able to know the purpose and its relationship with the content and will not be able to track the content easily.

**MAS Reference link:**
https://microsoft.sharepoint.com/:w:/s/accessibility/EZBde1C58eFKkdEvHy1sfB0BbfrE-sgqsVO5g2zVY6wbDA?e=jEWXd4

“Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/104483005/209672484-9c609e05-c4e3-4e84-b410-7376b64f8654.mp4

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is working fine, hence closing this issue.

![image](https://github.com/user-attachments/assets/d73055a1-2404-41a4-bd09-7df53bccc15b)
